smarts/sstudio/sumo2mesh.py

```diff
@@ -23,9 +23,12 @@
 from smarts.sstudio.types import MapSpec
 
 
-def generate_glb_from_sumo_network(sumo_net_file, out_glb_file):
-    map_spec = MapSpec(sumo_net_file)
-    road_network = SumoRoadNetwork.from_spec(map_spec)
+def generate_glb_from_sumo_network(sumo_net_file, out_glb_file, road_network=None):
```

Comment:
This was added specifically so that we don't create another instance of `road_network`. The function is only used in `cli/studio.py` which earlier tried to create two instances, one to initialize and load the map and other to create the glb file. 
```python
        sumo_road_network = None
        map_net = str(scenario_root / "map.net.xml")
        if not allow_offset_map or scenario.traffic_histories:
            from smarts.sstudio.types import MapSpec

            map_spec = MapSpec(map_net)
            sumo_road_network = SumoRoadNetwork.from_spec(
                map_spec, shift_to_origin=True
            )
        elif os.path.isfile(SumoRoadNetwork.shifted_net_file_path(map_net)):
            click.echo(
                "WARNING: {} already exists.  Remove it if you want to use unshifted/offset map.net.xml instead.".format(
                    SumoRoadNetwork.shifted_net_file_name
```
That seem redundant so i thought maybe we should pass the instance from earlier to generate the same file. But yes there is a chance of error here if user ends up passing `sumo_net_file` doesn't match the `road_network`.

Reply:
I could give a few alternatives here:

- Split into two methods
   - One that generates glb from a file location
   - One that generates from an existing road map
- Remove the `sumo_net_file` argument and explicitly require a `RoadNetwork` object

Reply:
I think I lean towards removing the `sumo_net_file` argument.

Reply:
If we do that, we wont be able to generate `glb` file explicitly from a provided `sumo_net_file ` i.e. can't 
 use `generate_glb_from_sumo_network.py` in standalone mode as it was created to do so.